### DISCUSSION TITLE
Promoting and raising awareness about Belgif REST Guidelines

### DISCUSSION THREAD

**Discussion by jpraet** [belgif/rest-guide#183] [General]
The introduction paragraph of the REST Guidelines states:

> This style guide for RESTful services is a collaborative effort by several Belgian government institutions, originally under the [G-Cloud](https://www.gcloud.belgium.be/) umbrella, before moving to [Belgif](https://www.belgif.be/), the Belgian Interoperability Framework. Its goal is to improve compatibility between RESTful services offered by government institutions or any other organization adopting these guidelines.

Could we list the institutions that adopt the Belgif REST Guidelines somewhere? Maybe not in the guide itself, but linking to a wiki page of the github repo? If we have a big list of institutions already implementing the guidelines, it may convince others to follow along in this standardization effort. Having such list also enables us to know early on in the analysis phase, when integrating with an API of another institution, if their API will adhere to the Belgif REST Guidelines or not.

Other ideas to raise awareness about the guidelines? Sometimes even within institutions that officially adopt the Belgif REST Guideline, the API developers are not always aware of the guidelines.

**Comment by smals-vea** [comment]
Other ideas to raise awareness about the guidelines? Sometimes even within institutions that officially adopt the Belgif REST Guideline, the API developers are not always aware of the guidelines.

-> la seule manière de s'assurer que les développeurs suivent les guidelines REST est que, au sein de chaque institution partenaire, un REST Board soit mis en place afin qu'il puisse valider les API et donner du support aux équipes de développement -> un processus fort du style "pas de validation REST alors pas de déploiement' .
De plus, il faut agir sur la communication tant pour communiquer les Update du REST guidelines que pour faire la promo du validateur des OpenAPI mis sur GitHub.

**Comment by JDMKSZ** [comment]
I think it's a good idea to have such a list, so that if persons from within those institutions who are unaware of the guidelines, can be pointed to the fact that they are on that list. I would also add to the list the contact point within the institution. This doesn't have to be a mail address or person, just a department or group within the institution.
